packages/stores/types.ts

```diff
@@ -165,6 +201,7 @@ type TClientStore = {
     switched: boolean;
     switch_broadcast: boolean;
     switchEndSignal: () => void;
+    states_list: TLocationList[];
```

Comment:
```suggestion
    states_list: StatesList;
```
🤔

Reply:
In addressdetails component it has TLocationList type which was Statestype and some other object was also added.So I have to use this type

Reply:
@yaswanth-deriv This file will be our source of truth for types, Ignore the usage, Later other devs will fix those TS errors, This type should be the correct one so if you are sure `StatesList` is the type then change it please 🙏🏻